homework-submission/my-app/src/App.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import './App.css';
+import TodoItems from './Components/createDynamicItems';
+import Button from './Components/button';
+
+class Todolist extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      items: [
+        {
+          name: 'REACT Lesson',
+          deadline: 23 - 3 - 2019,
+          key: Date.now(),
+        },
+        {
+          name: 'Wach movie',
+          deadline: 23 - 3 - 2019,
+          key: Date.now() + 1,
+        },
+        {
+          name: 'Football',
+          deadline: 23 - 3 - 2019,
+          key: Date.now() + 2,
+        },
+      ],
+    };
+    this.addItem = this.addItem.bind(this);
+    this.deleteItem = this.deleteItem.bind(this);
+  }
+
+  addItem(e) {
+    if (this._inputElement.value !== '') {
+      let newItem = {
+        name: this._inputElement.value,
+        key: Date.now(),
+      };
+      this.setState(PrevState => {
+        return {
+          items: PrevState.items.concat(newItem),
+        };
+      });
+    }
+    this._inputElement.value = ' ';
```

Comment:
in the condition you have `''` if empty string apply otherwise move on to the next line, 
here you are telling `this._inputElement.value` is equals to white space.